js_hw_4/task_02.js

```diff
@@ -0,0 +1,68 @@
+// 1. У вас есть массив названий пицц вашего конкурента. Создайте функцию, которая будет принимать ваш набор названий пицц (массив) 
+//   и возвращать только те, которых нет у конкурента (тоже массив). Если все ваши пиццы есть у конкурента - вернуть null
+//   Пиццы конкурента:
+//   const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai']
+  
+
+
+const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai'];
+function checkCompetitorsPizzas(myPizzaArr) {
```

Comment:
Function must be case insensitive

Reply:
Implemented by adding competitorPizzas.map(word => word.toLowerCase())